### PR TITLE
Ignore files when exporting package

### PR DESCRIPTION
**Pull request by simonschaufi**
This commit is part of a campaign to reduce the amount of data transferred to save global bandwidth and reduce the amount of CO2. See https://github.com/Codeception/Codeception/pull/5527 for more info.

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
+/.gitignore            export-ignore
+/.php_cs.dist          export-ignore
+/Dockerfile            export-ignore
 /Makefile              export-ignore
-/phpstan-baseline.neon export-ignore
 /phpstan.neon.dist     export-ignore
+/phpstan-baseline.neon export-ignore
 /phpunit.xml.dist      export-ignore
-/tests/                export-ignore
+/UPGRADING.md          export-ignore
```
Comment (gmponos) [inline .gitattributes:14]: IMHO I believe that upgrading guide is something similar to changelog and it should stay

**Comment by Nyholm** [comment]
Thank you. I see that you added `UPGRADING.md` and `Dockerfile`. 

Could you please remove `UPGRADING.md` and dont reorder the items. I know it may look weird, but they were actually ordered. =)

**Comment by simonschaufi** [comment]
You have the permission to change the pr yourself, so please go ahead and adjust it like you want.

**Comment by Nyholm** [comment]
If you don’t want to work on this I’ll close the PR. 

I’m happy to help you make a change to guzzle. I am not doing the work for you.

**Comment by simonschaufi** [comment]
better now? will you accept it like this?

File: .gitattributes [hunk 2]
```diff
@@ -1,10 +1,11 @@
-.editorconfig          export-ignore
-.gitattributes         export-ignore
+/.editorconfig         export-ignore
```
Comment (gmponos) [inline .gitattributes:1]: Sorry to be a pain.. but...

https://github.com/guzzle/guzzle/pull/2412#discussion_r352331347

Comment (simonschaufi) [reply .gitattributes:1]: ok, I agree if that was on purpose.

**Pull request merged**